dbcon/execplan/treenode.h

```diff
@@ -829,8 +836,16 @@ inline float TreeNode::getFloatVal()
             return (float)fResult.doubleVal;
 
         case CalpontSystemCatalog::DECIMAL:
+        case CalpontSystemCatalog::UDECIMAL:
```

Comment:
Did I get it right that previously we returned nothing for UDECIMAL?

Reply:
Yes, previously we were throwing an exception for UDECIMAL.